src/icrar/leap-accelerate/algorithm/cpu/PhaseRotate.cc

```diff
@@ -186,13 +186,13 @@ namespace cpu
         }
 
         Matrix<double> A = Matrix<double>(a1.size() + 1, icrar::ArrayMax(a1) + 1);
-        for(auto v : A)
+        for(double& v : A)
```

Comment:
Isn't there a `fill`-like operation that could be used instead of an explicit loop?

This could also be `auto &` instead of `double &`, then doesn't need to be changed again if you change the type.

Reply:
I know there is the std::array::fill() method but I think the only similar thing is the std::fill() function for other collections. Happy to change to `std::fill(A.begin(), A.end(), 0);`

When I reimplement MetaData with Eigen there is a special constructors like Eigen::Matrix::Zero() that I'll swap to.

Reply:
There's also an assignment operator: https://casacore.github.io/casacore/classcasacore_1_1Matrix.html#a9c44d638f1a937d71369df6cbdbf2eca, so I think it could also be `A = 0;`. Same for the `Vector` just below.